.travis.yml

```diff
@@ -201,6 +201,9 @@ script:
   - $make PLATFORM=ls-ls1021atwr
   - $make PLATFORM=ls-ls1021aqds
 
+  # Xilinx Zynq 7000
```

Comment:
Zynq-7000 is the processor family, and zc702 is the board name; there are also zc706 and ZED board:
http://china.xilinx.com/products/boards-and-kits/ek-z7-zc702-g.html